Client/Frontend/Strings.swift

```diff
@@ -875,13 +880,14 @@ extension String {
         public static let IntroWelcomeTitle = MZLocalizedString(
             "Onboarding.Welcome.Title.v106",
             tableName: nil,
-            value: "Welcome to an independent internet",
+            value: "Make Firefox your go-to browser",
             comment: "String used to describes the title of what Firefox is on the welcome onboarding page for 106 version in our Onboarding screens.")
         public static let IntroWelcomeDescription = MZLocalizedString(
             "Onboarding.Welcome.Description.v106",
             tableName: nil,
-            value: "Firefox puts people over profits and defends your privacy by default.",
+            value: "Firefox puts people over profits and defends your privacy as you browse. ",
             comment: "String used to describes the description of what Firefox is on the welcome onboarding page for 106 version in our Onboarding screens.")
+        public static let IntroLinkButtonTitle = MZLocalizedString("Onboarding.Welcome.Link.Action.v106", tableName: nil, value: "Learn more in our privacy notice", comment: "String used to describe the title of link button is on the welcome onboarding page for 106 version in our Onboarding screens.")
```

Comment:
- Please review indentation
- If this is added for v114, v106 doesn't really apply here in the comment and key name